examples/topology_samplers.rst

```diff
@@ -0,0 +1,124 @@
+.. _topology_samplers:
+
+=================================
+Working With Different Topologies
+=================================
+
+The examples shows how to construct software samplers with the same structure
+as the :term:`QPU`, and how to work with :term:`embedding`\s with different
+topologies.
+
+The code examples below will use the following imports:
+
+>>> import neal
+>>> import dimod
+>>> import dwave_networkx as dnx
+>>> import networkx as nx
+>>> import dwave.embedding
+...
+>>> from dwave.system import DWaveSampler, EmbeddingComposite
+
+Creating a Chimera Sampler
+--------------------------
+
+As detailed in :ref:`cpu`, you might want to use a classical solver while
+developing your code or writing tests. However, it is sometimes useful to
+work with a software solver that behaves more like a quantum computer.
+
+One of the key features of the quantum computer is the :term:`hardware graph`, which
```

Comment:
Unclear what you mean by this term: do you mean topology (Chimera, Pegasus) or are you referring to the "working graph" which defines that actual as opposed to the ideal topology graph?